dash/backend/src/modules/auth/controllers/non-redirectable-login.controller.ts

```diff
@@ -103,6 +104,14 @@ export class NonRedirectableLoginController {
     async localAuthLoginAction(@Body() loginDto: LoginDto): Promise<any> {
         this.prometheusV1Service.loginCounter.inc(1);
         const users: UserProfileDto[] = await this.userProfileService.loadUserByEmail(loginDto.userName);
+        // array
+        const sampleJson = ['banana', 'apple', 'peach'];
+        // message object const sampleJson = {label: 'Error creating webhook for cluster', data: { clusterId: 5 }};
+        // null const sampleJson = null;
+        // text with symbols const sampleJson = '09:36:29.597260967: Notice Unexpected connection to K8s API Server from container (command=python -u /app/sidecar.py pid=46515 k8s.ns=monitoring-system k8s.pod=prometheus-stack-grafana-7594dc6d95-cjfjj container=87d81d268f9c image=quay.io/kiwigrid/k8s-sidecar:1.22.0 connection=10.244.2.25:35680->10.0.0.1:443)';
+        console.log(sampleJson);
```

Comment:
Is this lefta round by accident?

Reply:
I gave it a test after I added the function. Might have missed to remove them. The lines will be removed.